src/toast/vis.py

```diff
@@ -400,20 +406,20 @@ def plot_healpix_maps(
     figsize = (12, 6)
     figdpi = 100
 
-    def plot_single(data, vmin, vmax, out, gnomrot=None):
+    def plot_single(data, vmin, vmax, out, gnomrot=None, reso=4.0, xsize=1000):
```

Comment:
You could also consider adding `**kwargs` to the signature here and pass them to the plotting command. Obviously not needed now.